src/interpolation/details/ArborX_InterpDetailsPolynomialBasis.hpp

```diff
@@ -114,7 +119,7 @@ KOKKOS_FUNCTION auto evaluatePolynomialBasis(Point const &p)
     // Cannot use structured binding with constexpr
     static constexpr auto slice_lengths_struct =
         polynomialBasisSliceLengths<DIM, Degree>();
-    static constexpr auto &slice_lengths = slice_lengths_struct.arr;
+    auto &slice_lengths = slice_lengths_struct.arr;
```

Comment:
What happened here?

Reply:
SYCL stated that `slice_lengths` (`cell_sizes` before) cannot be used because it is a non-const static. No idea if that was a quirk of C++ or intel's compiler. [Error](https://cloud.cees.ornl.gov/jenkins-ci/job/ArborX/job/PR-946/6/clang/source.b9734fa2-3c91-453a-9365-9030094d9656/#144).

Reply:
We could also just avoid this extra variable and spell out `slice_lengths_struct.arr` instead of `slice_lengths` below.